saul-examples/src/test/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/SpatialRoleLabeling/SpRLDataModelReaderTests.scala

```diff
@@ -0,0 +1,177 @@
+package edu.illinois.cs.cogcomp.saulexamples.nlp.SpatialRoleLabeling
```

Comment:
Do you think you can deploy your trained models? 
[Here are](https://github.com/IllinoisCogComp/saul/wiki/Deploying-models-to-the-maven-repository) some instructions. We have the same thing for [other projects](https://github.com/IllinoisCogComp/saul/blob/master/build.sbt#L105-L107) as well. 
Given the models deployed, you can add end-to-end tests for this classifier.